src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -0,0 +1,1507 @@
+            return Unsafe.As<Vector<T>, Vector<double>>(ref value);
+        }
+
+        [Intrinsic]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static unsafe Vector<T> Equals(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                T value = ScalarEquals(left.GetElement(index), right.GetElement(index)) ? GetAllBitsSetValue() : default;
+                result.SetElement(index, value);
+            }
+
+            return result;
+        }
+
+        [Intrinsic]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static unsafe Vector<T> LessThan(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                T value = ScalarLessThan(left.GetElement(index), right.GetElement(index)) ? GetAllBitsSetValue() : default;
+                result.SetElement(index, value);
+            }
+
+            return result;
+        }
+
+        [Intrinsic]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static unsafe Vector<T> GreaterThan(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                T value = ScalarGreaterThan(left.GetElement(index), right.GetElement(index)) ? GetAllBitsSetValue() : default;
+                result.SetElement(index, value);
+            }
+
+            return result;
+        }
+
+        [Intrinsic]
+        internal static Vector<T> GreaterThanOrEqual(Vector<T> left, Vector<T> right)
+        {
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                T value = ScalarGreaterThanOrEqual(left.GetElement(index), right.GetElement(index)) ? GetAllBitsSetValue() : default;
+                result.SetElement(index, value);
+            }
+
+            return result;
+        }
+
+        [Intrinsic]
+        internal static Vector<T> LessThanOrEqual(Vector<T> left, Vector<T> right)
+        {
+
+            Vector<T> result = default;
+
+            for (nint index = 0; index < Count; index++)
+            {
+                T value = ScalarLessThanOrEqual(left.GetElement(index), right.GetElement(index)) ? GetAllBitsSetValue() : default;
+                result.SetElement(index, value);
+            }
+
+            return result;
+        }
+
+        [Intrinsic]
+        internal static Vector<T> ConditionalSelect(Vector<T> condition, Vector<T> left, Vector<T> right)
+        {
+            ThrowHelper.ThrowForUnsupportedVectorBaseType<T>();
+            Vector<T> result = default;
+
+            result._00 = (left._00 & condition._00) | (right._00 & ~condition._00);
```

Comment:
This was doing `(left & condition) | (Vector.AndNot(right, condition))`, which themselves just did a direct op on the underlying `long` field.
I've simplified it to directly doing the relevant ops on the two long fields.